README.md

```diff
@@ -118,17 +118,17 @@ credential | Allow access to private git repositories | In the credential() func
 
 ### Structure
 
-TL;DR Making a new job? Make it look like [this one](https://github.com/edx/jenkins-job-dsl/blob/master/testeng/jobs/backupJenkins.groovy#L47) NOT like [this one](https://github.com/edx/jenkins-job-dsl/blob/master/dataeng/jobs/analytics/AggregateDailyTrackingLogs.groovy#L11).
+TL;DR Making a new job? Make it look like [this one](https://github.com/edx/jenkins-job-dsl/blob/312355c0568328b3d7bacbb00c2e94a6f30f01eb/testeng/jobs/backupJenkins.groovy#L47) NOT like [this one](https://github.com/edx/jenkins-job-dsl/blob/master/dataeng/jobs/analytics/AggregateDailyTrackingLogs.groovy#L11).
 
 There are two common ways to write jobs:
-1: The Build jenkins jobs are often flattened (as shown in [backupJenkins.groovy](https://github.com/edx/jenkins-job-dsl/blob/master/testeng/jobs/backupJenkins.groovy#L47) ) or if there are many jobs that are similar you can loop over a list of Maps like we do in [upgradePythonRequirements.groovy](https://github.com/edx/jenkins-job-dsl/blob/master/testeng/jobs/upgradePythonRequirements.groovy#L206). This has the benefit of keeping all the logic in one file and being easy to understand.
+1: The Build jenkins jobs are often flattened (as shown in [backupJenkins.groovy](https://github.com/edx/jenkins-job-dsl/blob/312355c0568328b3d7bacbb00c2e94a6f30f01eb/testeng/jobs/backupJenkins.groovy#L47) ) or if there are many jobs that are similar you can loop over a list of Maps like we do in [upgradePythonRequirements.groovy](https://github.com/edx/jenkins-job-dsl/blob/312355c0568328b3d7bacbb00c2e94a6f30f01eb/testeng/jobs/upgradePythonRequirements.groovy#L206). This has the benefit of keeping all the logic in one file and being easy to understand.
```

Comment:
Maybe we should refer to the fact that the build jenkins jobs have been deprecated/removed, even if the design pattern is still relevant?